packages/server/test/__snapshots__/course.integration.ts.snap

```diff
@@ -47,7 +47,6 @@ exports[`Course Integration GET /courses/:id gets all queues that are not disabl
       ],
       "lastName": "Person",
       "name": "User Person",
-      "password": null,
```

Comment:
I love that we literally tested to make sure our endpoints were returning passwords lmaoo